src/Sylius/Bundle/AttributeBundle/AttributeType/TextAttributeType.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\AttributeBundle\AttributeType;
+
+use Sylius\Component\Attribute\AttributeType\AttributeTypeInterface;
+use Sylius\Component\Attribute\Model\AttributeValueInterface;
+use Symfony\Component\Validator\Constraints\Length;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class TextAttributeType implements AttributeTypeInterface
+{
+    const TYPE = 'text';
```

Comment:
Some of the attributes types are used in other place to... Here you have [an example](https://github.com/Sylius/Sylius/pull/3608/files#diff-ef512835caf1006aa7cb64436adaeed0R208)